ui-v2/app/serializers/nspace.js

```diff
@@ -42,18 +42,11 @@ export default Serializer.extend({
   },
   respondForCreateRecord: function(respond, serialized, data) {
     return respond((headers, body) => {
-      // potentially this could be default functionality
-      // we are basically making the assumption here that if a create
-      // comes back with a record then we should prefer that over the local
-      // one seeing as it has come from the backend, could we potenitally lose
-      // data that is stored in the frontend here?
-      // if we were to do this as a default thing we'd have to use uid here
-      // which we don't know until its gone through the serializer after this
-      const item = this.store.peekRecord('nspace', body.Name);
-      if (item) {
-        this.store.unloadRecord(item);
-      }
-      return body;
+      // The data properties sent to be saved in the backend
+      // or the same ones that we receive back if its successfull
+      // therefore we can just ignore the result and avoid ember-data
+      // syncing problems
```

Comment:
Is it possible for the server to receive additional updates between create and response? (I imagine this would be very much an edge case if so).

Reply:
Yes definitely possible, but I think blocking queries would reload this into the UI (if you have them enabled of course). I think the other way around is more concerning - so if I load up the edit screen in the UI, then someone makes a change elsewhere and then after than I hit save in the UI.

Ideally we'd need `cas` for that, but I also have a half baked plan to use blocking queries here to warn you of backend changes on the edit pages, that's still a little way off though.